tests/verify_membackend.erl

```diff
@@ -252,35 +255,55 @@ get_remote_vnode_pid(Node) ->
                                all_vnodes, [riak_kv_vnode]),
     VNode.
 
-%% this is silly fragile
+%% @doc Crack open the VNode state record to find
+%% the hidden number of used bytes within the
+%% riak_kv_memory_backend:state record.
+-spec parse_regular_state_fun(integer()) -> fun().
+parse_regular_state_fun(Offset) ->
+    fun(X) ->
+        element(Offset, element(4, element(2, X)))
+    end.
+
+%% @doc Crack open the VNode state record to find
+%% the hidden number of used bytes within the
+%% riak_kv_memory_backend:state record for a multi-
+%% backend
+-spec parse_multi_state_fun(integer()) -> fun().
+parse_multi_state_fun(Offset) ->
+    fun(X) ->
+        element(
+            3, lists:nth(
+                1, element(
+                    2, element(
+                        Offset, element(
+                            4, element(2, X))))))
+    end.
+
+%% this is silly fragile and only works for Riak 2.0+
```

Comment:
Same comment as above - I think the solution might be to make the backends support a reporting mechanism rather than trying to parse some opaque data object.